bin/update-ips.bash

```diff
@@ -20,6 +20,9 @@ if [[ "${2}" == "apply" ]]; then
 fi
 has_diff=0
 
+#TODO: To be changed when decision made on networkpolicies for azure storage
+STORAGE_SERVICE=$(yq4 '.objectStorage.type' "${CK8S_CONFIG_PATH}/defaults/common-config.yaml")
```

Comment:
Make this lowercase to align with the rest of the script. Uppercase variables are primarily used to indicate variables that are in your environment.